# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/xemacs/no-shared.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -urN xemacs-21.4.12/Makefile.in.in xemacs-21.4.12-fixed/Makefile.in.in
--- xemacs-21.4.12/Makefile.in.in	2002-11-01 04:51:47.000000000 +0100
+++ xemacs-21.4.12-fixed/Makefile.in.in	2003-03-29 02:07:21.000000000 +0100
@@ -488,7 +488,7 @@
 	  done ; \
 	fi
 	cd ${srcdir}/etc && \
-	  for page in xemacs etags ctags gnuserv gnuclient gnuattach gnudoit; do \
+	  for page in xemacs ; do \
 	    ${INSTALL_DATA} ${srcdir}/etc/$${page}.1 ${mandir}/$${page}${manext} ; \
 	    chmod 0644 ${mandir}/$${page}${manext} ; \
 	  done
diff -urN xemacs-21.4.12/lib-src/Makefile.in.in xemacs-21.4.12-fixed/lib-src/Makefile.in.in
--- xemacs-21.4.12/lib-src/Makefile.in.in	2002-08-20 13:34:16.000000000 +0200
+++ xemacs-21.4.12-fixed/lib-src/Makefile.in.in	2003-03-29 02:00:12.000000000 +0100
@@ -253,13 +253,8 @@
 	fi
 
 install: ${archlibdir}
-	@echo; echo "Installing utilities for users to run."
-	for file in ${PUBLIC_INSTALLABLE_EXES} ; do \
-	  (cd .. && $(INSTALL_PROGRAM) lib-src/$${file} ${bindir}/$${file}) ; \
-	done
-	for file in ${PUBLIC_INSTALLABLE_SCRIPTS} ; do \
-	  (cd .. && $(INSTALL_PROGRAM) ${srcdir}/$${file} ${bindir}/$${file}) ; \
-	done
+	@echo; echo "Installing no utilities for users."
+	@echo; echo "They are shared with emacs and in a seperate ROCK package."
 
 uninstall:
 	cd ${bindir} && $(RM) ${PUBLIC_INSTALLABLES}
